packages/Search/src/details/DTK_DetailsTreeConstruction_decl.hpp

```diff
@@ -59,8 +62,7 @@ struct TreeConstruction
         Kokkos::View<Node *, DeviceType> internal_nodes );
 
     static void
-    calculateBoundingBoxes( Kokkos::View<Node *, DeviceType> leaf_nodes,
-                            Kokkos::View<Node *, DeviceType> internal_nodes );
+    calculateBoundingBoxes( BoundingVolumeHierarchy<DeviceType> bvh );
```

Comment:
Please add a comment on the new signature for this function.  Say that we need to pass something able to map a pointer to an internal node to its position within some sequential container that stores all internal nodes.
I really dislike this but I don't have better at this time.  Do you know what would be the cost associated to passing the two views and the root and some lambda that provide the mapping we need?

Reply:
Actually forget what I said.  Internal nodes are stored in a sequence container so it is valid to do `node - root` to figure out the index.
So do this instead
```C++
    calculateBoundingBoxes( Kokkos::View<Node *, DeviceType> leaf_nodes,
                             Node const *root );
```

Reply:
I need `bvh` in `calculateBoundingBoxes` to be able to use `getBoundingVolume()`.